kernel/core/os.adb

```diff
@@ -20,19 +20,19 @@ is
    subtype os_mbx_count_t is types.uint8_t range 0 .. OS_MAX_MBX_CNT;
 
    type os_mbx_t_array is
-              array (os_mbx_index_t) of aliased os_mbx_entry_t;
+              array (os_mbx_index_t) of os_mbx_entry_t;
```

Comment:
these are internal data (not shared with C/assembly) and therefore I guess we can remove the aliased attribute.